corehq/apps/export/views.py

```diff
@@ -2358,7 +2357,7 @@ def get_multimedia_task_kwargs(self, in_data, filter_form, export_object, downlo
     def prepare_custom_export(self, in_data):
         prepare_custom_export = super(DownloadNewFormExportView, self).prepare_custom_export(in_data)
 
-        if super(DownloadNewFormExportView, self).has_export_documents(in_data):
+        if super(DownloadNewFormExportView, self).check_if_export_has_data(in_data):
```

Comment:
Why do these use `super`?  Couldn't it just check
```python
if self.check_if_export_has_data(in_data):
```

Reply:
@esoergel you are right. I've fixed this. Thank you.